AppInspector.RulesEngine/RulesVerifier.cs

```diff
@@ -148,12 +148,27 @@ public RuleStatus CheckIntegrity(ConvertedOatRule convertedOatRule)
                     {
                         throw new ArgumentException();
                     }
-
+                    
                     RegexOptions regexOpts =
-                        Utils.RegexModifierToRegexOptions(searchPattern.Modifiers ?? Array.Empty<string>());
+                        Utils.RegexModifierToRegexOptions(searchPattern.Modifiers);
+                    
+#if NET7_0_OR_GREATER
```

Comment:
Same as above. This should be part of the conversion method.

Reply:
sure, additional optional parameter for RegexModifierToRegexOptions should work fine.